docs/content/PageLayout.mdx

```diff
@@ -5,6 +5,7 @@ status: Alpha
 # description: TODO
 source: https://github.com/primer/react/tree/main/src/PageLayout
 storybook: https://primer.style/react/storybook?path=/story/layout-pagelayout--default
+a11yReviewed: true
```

Comment:
💖 thank you for remembering to add this in! I have a PR open to add a task in the issue template as a reminder.